src/lib.rs

```diff
@@ -272,6 +272,12 @@ impl From<serde_json_core::de::Error> for Error {
     }
 }
 
+impl From<serde_json_core::ser::Error> for Error {
+    fn from(err: serde_json_core::ser::Error) -> Error {
+        Error::Serialization(err)
```

Comment:
Thanks! This is something I noticed recently too - we definitely don't want to just drop the serializer error

Reply:
Yes. It was fine up to now since there is only exactly one serializer error which is just buffer full as the comment says. But that;as certainly not to be relied on and problematic for #103